test/framework/integration_framework/fake_peer/network/loader_grpc.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef INTEGRATION_FRAMEWORK_FAKE_PEER_LOADER_GRPC_HPP_
+#define INTEGRATION_FRAMEWORK_FAKE_PEER_LOADER_GRPC_HPP_
+
+#include "framework/integration_framework/fake_peer/types.hpp"
+#include "loader.grpc.pb.h"
+#include "logger/logger.hpp"
+
+namespace integration_framework {
+  namespace fake_peer {
+
+    class LoaderGrpc : public iroha::network::proto::Loader::Service {
+     public:
+      explicit LoaderGrpc(const std::shared_ptr<FakePeer> &fake_peer);
+
+      grpc::Status retrieveBlocks(
+          ::grpc::ServerContext *context,
```

Comment:
Please confirm that raw pointers here and below are ok for use.

Should we use here smart pointers or const refs instead?

Reply:
It is ok as we implement the GRPC class interface.